src/search-by-term.test.js

```diff
@@ -0,0 +1,35 @@
+const documents = require('./documents.mock');
```

Comment:
File is called `documents` but exports an object of 1 document, maybe export an array?